packages/training/python/tests/test_training_orchestrator.py

```
@@ -36,15 +36,18 @@ def test_returns_list(self):
     def test_missing_database_url(self):
-        """Test error when DATABASE_URL not set"""
+        """Test error when DATABASE_URL not set (when using db source)"""
         original = os.environ.get("DATABASE_URL")
         if "DATABASE_URL" in os.environ:
             del os.environ["DATABASE_URL"]
         
         try:
             errors = validate_environment()
+            # Note: DATABASE_URL is only required when trajectory_source=db
+            # This may return empty if trajectory_source is huggingface
             db_errors = [e for e in errors if "DATABASE_URL" in e]
```

Ruff (0.14.14)

[error] Local variable `db_errors` is assigned to but never used

Remove assignment to unused variable `db_errors`

(F841)